src/extractor/graph_compressor.cpp

```diff
@@ -13,8 +13,7 @@ namespace osrm
 namespace extractor
 {
 
-GraphCompressor::GraphCompressor(SpeedProfileProperties speed_profile)
-    : speed_profile(std::move(speed_profile))
+GraphCompressor::GraphCompressor()
```

Comment:
This can be `GraphCompressor() = default;` in the header.
I can see this or the same with dtors all over the code base.